console/src/main/java/org/georchestra/console/ws/edituserdetails/EditUserDetailsFormController.java

```diff
@@ -133,7 +133,7 @@ private boolean isReferentOrSuperUser(Account userAccount) throws DataServiceExc
      *
      * @param account input data
      */
-    private EditUserDetailsFormBean createForm(final Account account) {
+    EditUserDetailsFormBean createForm(final Account account) {
```

Comment:
ok for package-private access (so that the method is testable), you could also make it public and use the `@VisibleForTesting` annotation (which is only meant for doc purposes anyway, and only if guava is available in the classpath, I am not sure if this is the case for the console)

Reply:
I tested both. Hard to make a decision but prefer this one in the end :man_shrugging: